docs/source/adodb.asciidoc

```diff
@@ -0,0 +1,161 @@
+== ADODB
+
+Sources with `type = "adodb"` configure ADODB sources.
+
+ADODB is supported on Windows only.
+Install https://github.com/mhammond/pywin32/releases[pywin32] separately to enable.
+The Timeseer installer includes pywin32 by default.
```

Comment:
Here it mentions Timeseer. It should probably be rephrased to not mention the product in the sources. (also in other places).